calcom/cal.com
# fix: Impersonate on users page

## What does this PR do?

Added an option as `Impersonate User` to admin/settings/user. 

Fixes https://github.com/calcom/cal.com/issues/13240
/claim #13240 

![image](https://github.com/calcom/cal.com/assets/58587256/e93f925a-9363-420a-8e56-bc6094820d7a)


## Type of change

- New feature (non-breaking change which adds functionality)

## How should this be tested?

- What is expected (happy path) to have (input and output)?

An option as `Impersonate User` is added to admin/settings/user. You can go to the url and click on the three dots on the right of the user information to view options. Click on "Impersonate User" to use the option.

## Mandatory Tasks

- [X ] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.
